tests/unit/__init__.py

```diff
@@ -1,6 +1,7 @@
+"""Ops testing settings."""
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
 import ops.testing
 
-ops.testing.SIMULATE_CAN_CONNECT = True
+ops.testing.SIMULATE_CAN_CONNECT = True  # type: ignore[attr-defined]
```

Comment:
You need to remove this. It's a breaking change in ops 2

Reply:
Oh man, thanks for spotting! Will remove it.